src/deva/env/dev/interface.py

```diff
@@ -49,21 +70,24 @@ def start(self) -> None:
         This method starts the developer environment. If this method returns early, the `status`
         method should contain information about the startup progress.
 
-        This method will never be called if the environment is already running.
+        This method will only be called if the environment is stopped or nonexistent.
         """
 
     @abstractmethod
     def stop(self) -> None:
         """
         This method stops the developer environment. If this method returns early, the `status`
         method should contain information about the shutdown progress.
+
+        This method will only be called if the environment is started.
```

Comment:
This is to avoid wasting resources. For example, say you don't want to re-create your environment because you are in the middle of something but you must pause the work to context switch or sleep. This would shut down the container or cloud resource but not remove it.

Reply:
I can see it for local env, less on cloud providers. But ok for `stop --remove`, we can always change the default